spec/services/collators/regular_income_collator_spec.rb

```diff
@@ -1,257 +1,257 @@
 require "rails_helper"
 
-RSpec.describe Collators::RegularIncomeCollator do
-  let(:assessment) { create(:assessment, :with_applicant, :with_gross_income_summary) }
-  let(:gross_income_summary) { assessment.gross_income_summary }
+# RSpec.describe Collators::RegularIncomeCollator do
```

Comment:
does this file need deleting or resurrecting?

Reply:
Resurrecting. Sorry, I don't even remember doing this!

Reply:
Oh, wait, no, deleting - there is no RegularIncomeCollator any more.